src/libraries/System.Text.Json/src/System.Text.Json.csproj

```diff
@@ -349,6 +349,7 @@ System.Text.Json.Utf8JsonReader</PackageDescription>
     <ProjectReference Include="$(LibrariesProjectRoot)Microsoft.Bcl.AsyncInterfaces\src\Microsoft.Bcl.AsyncInterfaces.csproj" />
   </ItemGroup>
   <ItemGroup>
-    <AnalyzerReference Include="..\gen\System.Text.Json.SourceGeneration.csproj" />
+    <AnalyzerReference Include="..\gen\System.Text.Json.SourceGeneration.3.9.csproj" />
```

Comment:
Similarly, consider using a `.Roslyn3.csproj` naming convention here.

Reply:
+1. Or `RoslynV3` as suggested elsewhere so it's clearer what the numbers mean.

Reply:
I chose to go with:
* System.Text.Json.SourceGeneration.Roslyn3.11.csproj
* System.Text.Json.SourceGeneration.Roslyn4.0.csproj
* System.Text.Json.SourceGeneration.Roslyn3.11.Unit.Tests.csproj
* etc